kalliope/signals/geolocation/geolocation.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from threading import Thread
+
+from kalliope.core import Utils
+from kalliope.core.ConfigurationManager import BrainLoader
+
+logging.basicConfig()
+logger = logging.getLogger("kalliope")
+
+
+class MissingParameter(Exception):
```

Comment:
LJust a note: like neuron, we should create a class dedicated to Signals Exceptions.

Reply:
Yep good idea, I will put all of them in a mother class